### ISSUE TITLE
Content takes up full textwidth

### ISSUE THREAD

**Issue by pranavats** [ox-tufte/ox-tufte#11]
Hi, when I publish the html file with `org-html-publish-to-tufte-html`, the content takes up the entire textwidth, and the sidenotes are ~~hidden~~ out of the screen on the right. (Something that should happen for a mobile device probably, but I'm on a 16:10 screen, firefox).

I use the following headers, with css from https://github.com/edwardtufte/tufte-css.

```org
#+HTML_HEAD: <link rel="stylesheet" type="text/css" href="./css/tufte.css" />
#+HTML_HEAD_EXTRA: <link rel="stylesheet" type="text/css" href="./css/latex.css" />
```
 I'm using ox-tufte from melpa, in spacemacs.

![test-file](https://user-images.githubusercontent.com/22037053/50749222-7ecdf580-1263-11e9-8f33-7b767d0e9176.png)

Source Code for generated HTML file: https://gist.github.com/pranavats/94e9fbd6077b1f2562cdb02812a63e0e

My `org-publish-project-alist`

```elisp
(require 'ox-publish)
(require 'ox-tufte)
(setq org-publish-project-alist
      '(
        ;;Dynamic Component: Handle org source files
        ("website-notes"
         :base-directory "./org/"
         :base-extension "org"
         :publishing-directory "./html/"
         :recursive t
         :publishing-function org-html-publish-to-tufte-html
         :headline-levels 4
         :auto-preamble t
         )

        ;;Static Component: Handle Non-Org Files
        ("website-static"
         :base-directory "./org/"
         :base-extension "css\\|js\\|png\\|jpg\\|gif\\|pdf\\|mp3\\|ogg\\|swf"
         :publishing-directory "./html/"
         :recursive t
         :publishing-function org-publish-attachment
         )

        ;;Publish/Dummy Component
        ("website":components ("website-notes" "website-static"))
        ))
```

**Comment by pranavats** [comment]
It is probably due to extra `iframe` added in the versions after v1.1. I am using v1.1 now, which works fine for at least 16:10 screens.

**Comment by codemac** [comment]
This is because of the <section> html tag not being used by the org exporter, I believe.

**Comment by josephkern** [comment]
Looks like there are some promising answers here about how to handled the margin and footnotes: https://jnboehm.gitlab.io/blog/tufte-css/

**Comment by mathiasp** [comment]
You can bring org-export to use section and article. Just configure your org-publish-alist with

	 :html-container "section"
	 :html-divs ((preamble "div" "preamble")
		     (content "article" "content")
		     (postamble "div" "postamble"))
	 :html-doctype "html5"
	 :headline-levels 2
	 :html-html5-fancy t

should do it. I have not used it extensively, just building up my blog, but it seems to work.

**Comment by Hallicopter** [comment]
How do I make this work? Where do I put these code snippets? (New to org and emacs)

**Comment by bayesians-inc-PL** [comment]
this will be solved ("out of the box") in the upcoming v2 release.

**Comment by bayesians-inc-PL** [comment]
the v2 release is now available for preview at https://github.com/ox-tufte/ox-tufte/tree/dev
for everyone's awareness, https://github.com/melpa/melpa/pull/8633 needs to be merged before `dev` branch can be merged into `main`
